pages/services/braids-and-twists/[service-id]/index.tsx

```diff
@@ -29,6 +29,9 @@ const BraidsAndTwistsService = ({ service }: Props) => {
     state.matches("selectExtensionUsage");
 
   useEffect(() => {
+    if (state.context?.service !== null && service.id !== state.context.service.id) {
```

Comment:
thanks Haiquan! Please let's add a comment as a reminder for why this if statement is there.. it will be nice to remember while looking at it, so we don't have to dig through our project board to remember. We can merge it right after.